packages/core/core/src/factories.ts

```diff
@@ -11,18 +11,6 @@ const symbols = {
 
 type WithStrapiCallback<T> = T | (<S extends { strapi: Strapi }>(params: S) => T);
 
-// Content type is proxied to allow for dynamic content type updates
```

Comment:
Here do we mean dynamic updates like adding the RW attributes when enabled? If
so, do we still need that ability?

Reply:
I refactored the review workflow code so we don't need to do that magic trick as this could have a lot of unexpected consequences.